.github/workflows/update_apis.yml

```
@@ -0,0 +1,24 @@
+name: AWS
+ on:
+  schedule:
+    - cron: '0 6 * * *'  # Daily at 6 a.m.
+
+jobs:
+  AWS:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: [1.4.0]
```

Could we just use 

```suggestion
        julia-version: [1] 
```

to get the latest stable release?